README.md

```diff
@@ -104,20 +105,21 @@ The following instructions all pertain to `provider/resources.go`, in the sectio
     "foo_something_else": {Tok: makeResource(mainMod, "SomethingElse")},
     ```
 
-1. **Add data source mappings:** For each data source in the provider, add an entry in the `DataSources` property of the `tfbridge.ProviderInfo`, e.g.:
+2. **Add data source mappings:** For each data source in the provider, add an entry in the `DataSources` property of the `tfbridge.ProviderInfo`, e.g.:
```

Comment:
My intent was to number them all with `1.` on purpose because MD will render these as `<li>` in a `<ul>`, which will automatically apply numbering.

Is there a reason to prefer 1, 2, 3, 4 over 1, 1, 1, 1, e.g. it's not rendering correctly somewhere?

Reply:
I think that's my editor trying to be smarter than me tbh: https://youtrack.jetbrains.com/issue/IDEA-274684

I am gonna have to use VSC for markdown I see :/